### ISSUE TITLE
Issue when creating a model that contains PK and FK

### ISSUE THREAD

**Issue by ardoramor** [LitePalFramework/LitePal#136]
I have a model that looks like:

```
public class WidgetRecord extends DataSupport {

    @Column(unique = true)
    private long id;

    @Column(nullable = false)
    private SoundRecord sound;
    
    ...
}
```

This causes an error when DB is created, stating that SQL encountered an error in statement `An exception that indicates there was an error with SQL parsing or execution. create table WidgetRecord (id integer primary key autoincrement,)`

I have to create a dummy field to fix it:

```
public class WidgetRecord extends DataSupport {

    @Column(unique = true)
    private long id;

    @Column(defaultValue = "true")
    private boolean dummy;

    @Column(nullable = false)
    private SoundRecord sound;
    
    ...
}
```

The error is because of the trailing comma after `autoincrement`. However, I may not be using the framework properly. Why doesn't the SQL contain something for the foreign key like `id_SoundRecord integer FOREIGN KEY REFERENCES SoundRecord(id)`?

Also, is there a way to force `id` to be of type `long`?

**Comment by guolindev** [comment]
Yep. This might be a problem. I'll try to fix it in the next release. 
And also, you can use long or int as field type of id as you want. But in SQLite they will both be treated as integer type for table column.

**Comment by guolindev** [comment]
Now it's fixed in 1.5.0 version.
